modules/administration/pages/reports.adoc

```diff
@@ -23,68 +34,94 @@ To get the report in CSV format, run this command at the command prompt on the s
 spacewalk-report <report_name>
 ----
 
+[[spacewalk-report-reporting-db]]
+== ``spacewalk-report`` and the reporting database
+
+[command]``spacewalk-report`` uses by default the new reporting database to extract the data.
+
+This means that the new generated reports will have some differences in the structure and the format of the data.
+
+The differences common to all reports are:
+
+- the report data is not changing in real-time, but it's updated only by the execution of a scheduled task;
+- data duplication has been removed and columns that were previously considered "multival" contains now multiple values separated by ``;``. This also means that the command line options ``--multival-on-rows`` and ``--multival-separator`` are no longer applicable to the new reports, as their behavior is now the default;
+- in all reports the new columns ``mgm_id`` and ``synced_date`` have been introduced to identify respectively the management server in the hub scenario and the last time the information was updated from the application database;
```

Comment:
```suggestion
- in all reports the new columns ``mgm_id`` and ``synced_date`` have been introduced to identify the management server in the hub scenario and the last time the information was updated from the application database;
```
I could be wrong but I think "respectively" is the wrong word here.  Do you mean "either ... or" or both at the same time?